app/assets/javascripts/shopping_lists.js

```diff
@@ -65,13 +64,13 @@ function shoppingListItemToggler() {
         let itemHtml = item.parentNode;
 
         if ( item.classList.contains('js-toggle') ) {
-          itemHtml.insertAdjacentHTML('afterbegin', '<i class="fas fa-check text-success"></i>')
+          itemHtml.insertAdjacentHTML('afterbegin', '<i class="text-success">&#10003;</i>')
```

Comment:
I swapped out the font awesome checkmark and put in a standard html checkmark in its place. I don't love this solution because it is not using the component class, but it seemed like the easiest solution given that we're inside of javascript here and the goal of the PR is to move away from font awesome.